roles/sap_vm_provision/PLATFORM_GUIDANCE.md

```diff
@@ -36,8 +36,9 @@ See below for the drop-down list of required environment resources on an Infrast
 - Compute Router
     - SNAT
```

Comment:
`SNAT` is the Cloud NAT attached to the Compute Router, I just used the generic name instead > https://cloud.google.com/nat/docs/gce-example#create-nat

Can re-write this line as `Cloud NAT (SNAT)` if clearer?

Reply:
Updating, I will commit it with HA fixes.